packages/web/src/components/track/GiantTrackTile.tsx

```diff
@@ -106,6 +108,32 @@ const messages = {
     `Releases ${formatReleaseDate({ date: releaseDate, withHour: true })}`
 }
 
+const renderMood = (mood: Mood) => {
+  return (
+    <TextLink
+      to={{
+        pathname: generatePath(SEARCH_PAGE, { category: 'tracks' }),
+        search: new URLSearchParams({ mood }).toString()
```

Comment:
It feels odd to create a new instance on every render, but it's probably fine.

Reply:
this was something i copied over. good point tho. cc @Kyle-Shanks

Reply:
ya sounds good to me